src/org/opendatakit/briefcase/export/ExportConfiguration.java

```diff
@@ -77,18 +83,28 @@ public static ExportConfiguration load(BriefcasePreferences prefs) {
         prefs.nullSafeGet(START_DATE).map(LocalDate::parse),
         prefs.nullSafeGet(END_DATE).map(LocalDate::parse),
         prefs.nullSafeGet(PULL_BEFORE).map(Boolean::valueOf),
-        prefs.nullSafeGet(PULL_BEFORE_OVERRIDE).map(PullBeforeOverrideOption::from)
+        prefs.nullSafeGet(PULL_BEFORE_OVERRIDE).map(PullBeforeOverrideOption::from),
+        prefs.nullSafeGet(FORM_NEEDS_PRIVATE_KEY).map(Boolean::valueOf)
     );
   }
 
-  public static ExportConfiguration load(BriefcasePreferences prefs, String keyPrefix) {
+  public static ExportConfiguration load(Optional<FormStatus> form, BriefcasePreferences prefs, String keyPrefix) {
```

Comment:
This is only used by `ExportForms.load` and we will always have a form to pass to `ExportConfiguration.load`. I think you can safely change the type of the `form` argument to `FormStatus` instead of `Optional<FormStatus>`.

Also, let's change the name to explicitly state that this factory is for loading override configuration for forms.